arm-recoveryservices/2016-06-01/swagger/vaults.json

```diff
@@ -615,8 +684,7 @@
       ],
       "properties": {
         "properties": {
-          "$ref": "#/definitions/VaultProperties",
-          "x-ms-client-flatten": true
+          "$ref": "#/definitions/VaultProperties"

```

Comment:
Yes, flattening is recommended. If there was a bug with flattening applied, it would be very helpful for us to *file that bug* (tons of services rely on flattening - so it better be bug free).